index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header-content">
+      <div class="sub-header-container">
+        <div class="logo">
+          <a href="#"><img src="./asset/pascal-pasc-logo.png" alt="Logo" /></a>
+        </div>
+        <nav class="bars-icon">
+          <span class="bars"><i class="fa fa-bars"></i></span>
+        </nav>
+      </div>
+    </header>
+    <section>
+      <main class="main-container">
```

Comment:
- [ ] The `“main”` element must not appear as a descendant of the `“section”` element, Please try to fix that, that will improve your projects to look more semantic (check this [link](https://www.w3.org/TR/2015/WD-html51-20150506/semantics.html) for more  infos). I am sure that well be the reason too of your linters errors,